lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -341,12 +343,42 @@ newStakePoolLayer tr db@DBLayer{..} nl metadataDir = StakePoolLayer
                     , performance
                     , cost = poolCost
                     , margin = poolMargin
+                    , desirability =
+                        Ranking.desirability
+                            jormungandrITNConstants
+                            (Ranking.Pool
+                                (Ranking.unsafeMkRatio 0) -- pool leader pledge
+                                (Ranking.Lovelace $ fromIntegral $ getQuantity poolCost)
+                                (Ranking.unsafeMkRatio $ fromIntegral (getPercentage poolMargin) / 100)
+                                (Ranking.unsafeMkNonNegative performance))
                     }
                 , poolOwners
                 )
 
-    sortByPerformance :: [(StakePool, a)] -> [(StakePool, a)]
-    sortByPerformance = sortOn (Down . performance . fst)
+    -- | Constants needed for ranking pools. Specific to the incentiviced
+    -- testnet.
+    jormungandrITNConstants :: Ranking.EpochConstants
+    jormungandrITNConstants = Ranking.EpochConstants
+        { Ranking.leaderStakeInfluence = Ranking.unsafeMkNonNegative 0
+        , Ranking.desiredNumberOfPools = Ranking.unsafeMkPositive 100
+        , Ranking.totalRewards = Ranking.Lovelace $ deductTreasuryTax 3835616440000
+          -- TODO: take rewardDrawingLimitMax into account (depends on stake)
```

Comment:
We should be able to get the total active stake from the map and compare it with a hard-coded value.

Unless a lot of stake is undelegated, it should not have any effect. 

Without being absolutely sure we're on the same page as jormungandr, even this simple thing seems like unnecessary risk for the next release, which we could do without.

Reply:
> Unless a lot of stake is undelegated, it should not have any effect.

That is the case. ~85% of the stake is not delegated at the moment.

Reply:
> That is the case. ~85% of the stake is not delegated at the moment

I believed it looked at the "active stake". But a double check would be good.